### DISCUSSION TITLE
The number of region with the motif sequence is different between HOMER outputs

### DISCUSSION THREAD

**Discussion by h20gg702** [sipcapture/homer#644] [Q&A]
Hi, I conducted the motif enrichment analysis by HOMER findMotifsGenome.pl. Then I try to get the location where the motif is enriched by following the command (http://homer.ucsd.edu/homer/ngs/peakMotifs.html).
`findMotifsGenome.pl \
"region.hb" \
hg38 \
"/output_file_path/" \
-size 250 -p 8`
And then I used following command with output motif.motif by above command.
`findMotifsGenome.pl \
"region.hb" \
hg38 \
"/temp_path/" \
-find '/known1.motif' > '/known1_output.txt'`
I also tryed the other option HOMER sugget.
`annotatePeaks.pl \
"region.hb" \
hg38 \
-m '/known1.motif' > '/known1_output2.txt'`
My problem is that the number of region with the motif sequence is different between these above three output files. First output said that # of Target Sequences with Motif(of 124516) is 11966. 2nd output has 124524 rows and 2435 rows having Motif Instances. last output has 10201 rows. Does anyone knows which file is saying correct number of regions having the enriched motif? Thanks in advance.

**Comment by lmangani** [comment]
Wrong homer
